<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scannerClass</class>
 <widget class="QMainWindow" name="scannerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>scanner</string>
  </property>
  <property name="windowIcon">
   <iconset resource="scanner.qrc">
    <normaloff>:/scanner/Resources/Icons/3D.jpg</normaloff>:/scanner/Resources/Icons/3D.jpg</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QVTKWidget" name="qvtkWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>view</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>setting</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuSetting"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>240</width>
     <height>100</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Stereo Reconstruct</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButtonStereoReconstruct">
       <property name="text">
        <string>Stereo Reconstruct</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetProperties">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>111</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>240</width>
     <height>524287</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContentsProperty"/>
  </widget>
  <widget class="QDockWidget" name="dockWidgetActorManager">
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>100</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Actor Manager</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContentsActorManager"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="scanner.qrc"/>
 </resources>
 <connections/>
</ui>
